### DISCUSSION TITLE
Mainstage -> Quicktime

### DISCUSSION THREAD

**Discussion by benbravo1** [ExistentialAudio/BlackHole#88] [General]
Hi,

Blackhole seems like a great alternative to JackOSX that I used a lot. Unfortunately, I am totally unable to get any monitoring as soon as I send audio to it. I have tried multi-output devices, aggregates, etc., just like described, nothing does it. I get my sound sent to the target app (for example QuickTime from Logic), but it is impossible to hear what I am doing as soon as audio is sent to BlackHole. Recording is possible and works like it should. 

I am on Mac OSX 10.14.6

Thanks for you help

**Comment by devinroth** [comment]
Send me some screenshots of how you have everything setup and I may be able to help you out.

**Comment by benbravo1** [comment]
thanks for the quick reply!

Here's my setup:

1. Mainstage piano into blackhole
<img width="566" alt="Screenshot 2020-04-01 at 18 15 05" src="https://user-images.githubusercontent.com/63004069/78160795-bca8d200-7444-11ea-8ed7-d855ec562a2a.png">

2. Blackhole into Quicktime
<img width="380" alt="Screenshot 2020-04-01 at 18 16 36" src="https://user-images.githubusercontent.com/63004069/78160954-ff6aaa00-7444-11ea-92f8-d162e0da64ad.png">

Sound is coming in QuickTime alright. How do I hear what I am playing?

I've got a multi-output set up like so:
<img width="886" alt="Screenshot 2020-04-01 at 18 18 08" src="https://user-images.githubusercontent.com/63004069/78161106-4062be80-7445-11ea-958a-5f78cf434ee1.png">

Nothing is coming out when I choose the multi-output as my audio output, neither blackhole nor aggregate device...
<img width="205" alt="Screenshot 2020-04-01 at 18 19 25" src="https://user-images.githubusercontent.com/63004069/78161161-52dcf800-7445-11ea-868d-33b19e675c2b.png">

**Comment by devinroth** [comment]
Set MainStage output to the multi-output. I think that should solve your problem.

**Comment by benbravo1** [comment]
yes it does. I had a wrong approach I guess. I'll try it this way. thanks for your quick and efficient help

**Comment by devinroth** [comment]
You’re welcome.
